magefiles/magefile.go

```diff
@@ -491,6 +486,11 @@ func (ci CI) setRequiredEnvVars() error {
 				return err
 			}
 			os.Setenv("E2E_TEST_SUITE_LABEL", "e2e-demo,rhtap-demo,spi-suite,remote-secret,integration-service,ec,byoc,build-templates,multi-platform")
+		} else if openshiftJobSpec.Refs.Repo == "release-service-catalog" {
```

Comment:
I'm not really sure if the program ever gets to this condition. Doesn't it evaluate [this condition as true](https://github.com/redhat-appstudio/e2e-tests/pull/957/files#diff-d7a1577b2d546e34aaf816bdd1b83edef28a66a555f049767bafa83268b7efbbR330), so it skips the one on this line?

Reply:
yeah good point

Reply:
Update the condition to `HasSuffix` which should fix this and not break any existing cases